torch/csrc/jit/passes/peephole.cpp

```diff
@@ -41,6 +41,18 @@ void PeepholeOptimize(Block * block) {
           // Let DCE clean up any unused nodes at this point
         }
       } break;
+      case aten::type_as: {
+        if (n->inputs().size() != 2) {
```

Comment:
An assertion might be better here (is it possible for `aten::type_as` to have anything other than 2 arguments?)